From f53e79acd0d42f5c41e1104056af9cf23e9e2e2a Mon Sep 17 00:00:00 2001
From: "Andreas K. Huettel (dilfridge)" <dilfridge@gentoo.org>
Date: Sun, 4 Jan 2015 15:33:35 +0100
Subject: [PATCH] Replace bash-specific code from configure.ac, fixes build for
 sh=dash users
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Original patch by Rémi Cardona <remi@gentoo.org>

Change-Id: I815567df8a868c3997e8ac357b8529fded28c950
Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=525454
Reviewed-on: https://gerrit.libreoffice.org/13731
Reviewed-by: Caolán McNamara <caolanm@redhat.com>
Tested-by: Caolán McNamara <caolanm@redhat.com>
---
 configure.ac | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8aa33d0..5dbe640 100644
--- a/configure.ac
+++ b/configure.ac
@@ -11483,9 +11483,8 @@ test_font_map()
     FONTFILE_LOWER="$(echo $FONTFILE | $AWK '{print tolower($0)}')"
 
     TESTEXPR="'${FONTFILE_LOWER}' = '$(echo $FONT | $AWK '{print tolower($0)}').ttf'"
-    while true ; do
+    while test "$#" -ge 1 ; do
         MAPPING="$(echo $1 | $AWK '{print tolower($0)}')"; shift
-        test -n "$MAPPING" || break
         TESTEXPR="${TESTEXPR} -o '${FONTFILE_LOWER}' = '$MAPPING-regular.ttf'"
     done
     if test $TESTEXPR
-- 
2.3.0

